.github/workflows/publish-nuget.yaml

```diff
@@ -0,0 +1,56 @@
+name: publish
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - "*"
+
+jobs:
+  build-and-publish-nuget:
+    name: build and publish nuget
+    runs-on: windows-latest
```

Comment:
We should probably run this on our own runners. Github-hosted runners are very expensive, especially anything other than Linux. This would need to wait until our Windows runners have been migrated to organization-level, rather than repository-level. Currently they're tied to the `friday` repository.

Reply:
Since this is a public repository, this should be free, right?